src/main/java/eu/mrndesign/matned/jsonplaceholder/exception/PostNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package eu.mrndesign.matned.jsonplaceholder.exception;
+
+
+public class PostNotFoundException extends RuntimeException{
```

Comment:
W przypadku otrzymania tego błędu serwer zwraca 500. Najlepiej aby to było 404 albo 422 z odpowiednim komunikatem